docker-for-windows/index.md

```diff
@@ -443,64 +441,66 @@ to change context so that `kubectl` is pointing to `docker-for-desktop`:
+  visible, select **Show system containers (advanced)** and click **Apply**.
   Most users do not need this option.
 
 - To disable Kubernetes support at any time, deselect **Enable Kubernetes**.
   The Kubernetes containers are stopped and removed, and the
   `/usr/local/bin/kubectl` command is removed.
 
-  For more about using the Kubernetes integration with Docker Desktop for Windows,
-  see [Deploy on Kubernetes](kubernetes.md).
+  For more about using the Kubernetes integration with Docker Desktop, see [Deploy on Kubernetes](kubernetes.md).
 
 ### Reset
 
-On the Reset tab, you can restart Docker or reset its configuration.
+> Reset and Restart options
+>
+> On Edge releases, the **Restart Docker Desktop**, **Reset Kubernetes Cluster**, and **Reset to factory defaults** options are available on the **Troubleshoot** menu.
+
+On Stable releases, you can restart Docker or reset its configuration using the Reset tab.
 
 ![Reset](images/settings-reset.png){:width="600px"}
 
-* **Restart Docker** - Shuts down and restarts the Docker application.
+
+* **Restart Docker Desktop** - Shuts down and restarts the Docker application.
+
+* **Reset Kubernetes Cluster** - Deletes all stacks and Kubernetes resources.
 
 * **Reset to factory defaults** - Resets Docker to factory defaults. This is
   useful in cases where Docker stops working or becomes unresponsive.
 
-### Diagnose & feedback
+### Troubleshoot
 
 Visit our [Logs and Troubleshooting](troubleshoot.md) guide for more details.
 
 Log on to our [Docker Desktop for Windows forum](https://forums.docker.com/c/docker-for-windows) to get help from the community, review current user topics, or join a discussion.
 
 Log on to [Docker Desktop for Windows issues on GitHub](https://github.com/docker/for-win/issues) to report bugs or problems and review community reported issues.
 
-To give feedback on the documentation or update it yourself, use the Feedback options at the bottom of each docs page.
+For information about providing feedback on the documentation or update it yourself, see [Contribute to documentation](/opensource/).
 
 ## Switch between Windows and Linux containers
 
-From the Docker Desktop for Windows menu, you can toggle which daemon (Linux or Windows)
+From the Docker Desktop menu, you can toggle which daemon (Linux or Windows)
 the Docker CLI talks to. Select **Switch to Windows containers** to use Windows
 containers, or select **Switch to Linux containers** to use Linux containers
 (the default).
 
-![Windows-Linux container types switch](images/docker-menu-switch.png){:width="400px"}
+![Windows-Linux container types switch](images/docker-menu-switch.png){:width="300px"}
 
-For more information on Windows containers, refer to this documentation:
+For more information on Windows containers, refer to the following documentation:
 
 - Microsoft documentation on [Windows containers](https://docs.microsoft.com/en-us/virtualization/windowscontainers/about/index).
```

Comment:
Can you check the formatting here? It looks like this markdown to create a list, but there is only one item in the list.

Reply:
It looks correct in the Netlify preview - https://deploy-preview-9256--docsdocker.netlify.com/docker-for-windows/#switch-between-windows-and-linux-containers.

Let me know if I'm missing something.